server/handle.js

```diff
@@ -8,7 +8,7 @@ const fromError = require('./fromError')
 const calm = curry((next, data) =>
   Promise.resolve(data)
     .then(next)
-    .catch(identity)
+    .catch(console.error.bind(console))
```

Comment:
No need to bind `console.error` in node.  It's already bound.  Also, we need to `JSON.stringify` them somehow, since `loggly` needs that to parse properly.  I fiddled around in a node repl, and this works and keeps the stack:
```js
const logError = compose(console.error, JSON.stringify, pick(['message', 'name', 'stack']))

const calm = curry((next, data) =>
  Promise.resolve(data)
    .then(next)
    .catch(logError)
)
```

Reply:
Hoisting `console.error` into an outer `logError` function has the unfortunate side effect of making `console.error` significantly more difficult to spy on. I counted my losses & [kept `console.error` in a closure](https://github.com/articulate/sox/pull/2/commits/583bb76b9da381541a54cc72252aba88525ce628#diff-d4b32bf572a67671f77a4c822cfcde35R14).